ygot/schema_tests/schema_test.go

```diff
@@ -187,9 +187,6 @@ func TestDiff(t *testing.T) {
 		}(),
 		want: &gnmipb.Notification{
 			Update: []*gnmipb.Update{{
-				Path: mustPath("neighbors/neighbor[neighbor-address=192.0.2.1]/neighbor-address"),
```

Comment:
This isn't right -- I think related to the above problem, this should be produced.